packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -171,7 +171,7 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
         rel={ rel }
         aria-label={ link.ariaLabel }
       >
-        { link.name }
+        { this.props.onRenderLink!(link) }
```

Comment:
Wow, this is an old implementation of what we're now doing with IRenderFunction. Are you up for getting this render function in line with current practice.

1. change props to use IRenderFunction
2. move default render to private function _onRenderLink
3. change prop to onRenderLink = this._onRenderLink
4. change line 174 to { onRenderLink(link, this._onRenderLink) }